src/utils/convertFromSecondToMinutes.js

```diff
@@ -0,0 +1,7 @@
+const convertFromSecondToMinutes = (time) => {
+    const min = Math.floor(time / 60);
+    const sec = (time % 60).toString().padStart(2, '0');
+    return `${min} : ${sec}`;
```

Comment:
we should remove the spaces here, and if we need to display them apart we can do in the component, where the presentation logic is.